test.py

```
@@ -0,0 +1,17 @@
+def greet(name):
+    print(f"Hello, {name}!")
+
+def calculate_sum(a, b):
+    return a + b
+
+def main():
```

Ruff

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)